sdkproject/Assets/Mapbox/Unity/MeshGeneration/Modifiers/MeshModifiers/UvModifier.cs

```diff
@@ -38,8 +38,12 @@ public override void SetProperties(ModifierProperties properties)
 
 		public override void Run(VectorFeatureUnity feature, MeshData md, UnityTile tile = null)
 		{
+			if (md.Vertices.Count == 0 || feature == null || feature.Points.Count < 1)
```

Comment:
Please use braces even for single line code blocks.
https://github.com/mapbox/mapbox-unity-sdk/blob/develop/CODING-STYLE.md